CHANGELOG.md

```diff
@@ -108,7 +108,8 @@ Bug Fixes and Improvements
 - Fixed a bug in rendering profiles that involve two separate shifts. (#645)
 - Fixed a bug if drawImage was given odd nx, ny parameters, the drawn profile
   was not correctly centered in the image. (#645)
-
+- Added intermediate results cache to `ChromaticObject.drawImage` and
```

Comment:
I'm not sure this is enough to say what the purpose of the change was; include the fact that this can speed up rendering by such-and-such factor?